atomic_reactor/schemas/container.json

```diff
@@ -67,6 +67,15 @@
       "type": "integer",
       "minimum": 1,
       "default": 1
+    },
+    "go": {
+      "type": ["object", "null"],
+      "properties": {
+        "module": {
+          "type": "string"
+        }
+      },
+      "additionalProperties": true
```

Comment:
Please don't include this part... I will update #1159 instead. But good catch about additionalProperties, I had forgotten to put it in.

Reply:
Oh, I had forgotten the other commit would deal with the schema. I will remove the changes in this file and wait for #1159 to be merged.

Reply:
Done